src/types.ts

```diff
@@ -97,8 +97,7 @@ export type Validation = {
  * @public
  */
 export type ConditionalValidation = {
-  predicate: (args: ValidationArgs) => boolean | Promise<boolean>
-  message: (args: ValidationArgs) => string
```

Comment:
why is `message` deleted?

Reply:
because it needs to come from three lines above to not lose the error messages!